inhibit/inhibit.go

```diff
@@ -117,11 +117,11 @@ func (ih *Inhibitor) Mutes(lset model.LabelSet) bool {
 
 	for _, r := range ih.rules {
 		if r.TargetMatchers.Match(lset) && r.hasEqual(lset) {
-			ih.marker.SetInhibited(fp, true)
+			ih.marker.SetStatus(fp, types.Inhibited)
 			return true
 		}
 	}
-	ih.marker.SetInhibited(fp, false)
+	ih.marker.SetStatus(fp, types.Active)
```

Comment:
I'm not sure if this is semantically correct. It appears that alerts will be marked in the notify pipeline, so setting them to active here would be temporary until they get checked as part of the pipeline. Can someone confirm? @brancz